### DISCUSSION TITLE
eslint gives no meaningful feedback

### DISCUSSION THREAD

**Discussion by Legends** [eslint/eslint#17172] [Q&A] [answered]
In my package.json file i have added the following lint command:
```
 "scripts": {
    "li":"eslint . -c .eslintrc.js"
  }
```

Execute: `npm run li`

**Output:**
![image](https://github.com/eslint/eslint/assets/966368/c6481337-426e-44b2-a5f7-9bfc026fa1ad)

Where are the errors? is everything ok?

**Comment by Rec0iL99** [comment, marked as answer]
Hi @Legends, ESLint is working as expected. This just means ESLint has found no problems with your code.

If you want to check if ESLint is working, you can add a rule to `rules` in `.eslintrc.js`. 

```js
"no-unused-vars": "error"
```

```js
// somewhere in code
const foo = 1;
```

Now ESLint should report the `no-unused-vars` rule when you run the `li` script.

**Reply by Legends** [reply]
Thanks for also providing a check!

When doing the check with `unused-vars` I get the following result:

> C:\Users\User\Downloads\project\background.js
  141:7  error  'foo' is assigned a value but never used   no-unused-vars
  234:7  error  'foo2' is assigned a value but never used  no-unused-vars
 ✖ 2 problems (2 errors, 0 warnings)


Now I can click on the file, but is there a way to jump directly to the error found by clicking on the respective error?

**Reply by Rec0iL99** [reply]
If you are using the terminal in VSCode, then you can `cmd+click`/`ctrl+click` on `C:\Users\User\Downloads\project\background.js` in the terminal and it should take you to the error in the file.

I think it depends on the terminal client. I use iterm2 with Oh My Zsh, so it takes me directly to the error in the file in VSCode.

**Reply by Legends** [reply]
I know, it will take only take me to the first error, but for all subsequent errors I have to look up each via line/column position.
I also have installed the Eslint extension two weeks ago, but it did not show anything.
Now I have restartet VsCode and now I get EsLint errors as I type under "Problems" section and the respective errors are clickable, like i wanted.

So everything is fine now ;-)

Very strange...
